extra/3-magic-8-ball.js

```diff
@@ -68,7 +115,7 @@ function checkAnswer(answer) {
 
 There are some Tests in this file that will help you work out if your code is working.
 
-To run the tests for just this one file, type `npm test -- --testPathPattern 3-magic-8-ball` into your terminal
+To run these tests type `npm run extraTo run the tests for just this one file, type `npm run extra-tests -- --testPathPattern 3-magic-8-ball` into your terminal
```

Comment:
You need to change the package.json if you want to run all tests. See Marina's [work](https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week1/pull/230/files#diff-570b1756592e58b70ffde559c761a33651d2dec0eab8e3b18a59975605243b9f) for the changes on the 3-magic-8-ball.js and package.json